_partials/self-hosted/_setup-steps.mdx

```diff
@@ -0,0 +1,243 @@
+    warning: /opt/spectro/pwgen-2.08-3.el8.x86_64.rpm: Header V3 RSA/SHA256 Signature, key ID 2f86d6a1: NOKEY
+    Verifying...                          ################################# [100%]
+    Preparing...                          ################################# [100%]
+    Updating / installing...
+    1:pwgen-2.08-3.el8                 ################################# [100%]
+    Skipping setting up Nginx and Podman for airgap
+    Installing Podman
+    Setting up Harbor
+    Setting up oras and jq
+    Setting up Manifests
+    Cleaning up setup files
+    Reboot the system for selinux changes to take effect
+    ```
+
+14. Reboot your RHEL VM to apply the changes.
+
+    ```shell
+    reboot
+    ```
+
+    <!-- prettier-ignore -->
+    Your SSH connection will be terminated. Start a new SSH session and switch to `sudo` mode before proceeding. Use
+    the same commands from steps **5**, **6**, and **12** of this guide.
+
+15. Issue the following command to configure the Harbor registry. Replace `palette.example.com` with the FQDN of the
+    RHEL airgap VM. The script will generate a self-signed certificate for the value you provide.
+
+    ```shell
+    /bin/airgap-setup.sh palette.example.com
+    ```
+
+    The script output should look similar to the example below. It contains the credentials and values you will need when
+    completing the installation with the Palette CLI. If you need to review this information in the future, invoke the
+    script again.
+
+    ```text hideClipboard
+    Setting up SSL Certs
+    /opt/spectro/functions.sh: line 118: /etc/nginx/.htpasswd: No such file or directory
+    chmod: cannot access '/etc/nginx/.htpasswd': No such file or directory
+    mkdir: cannot create directory ‘/etc/nginx/ssl’: No such file or directory
+    cp: target '/etc/nginx/ssl' is not a directory
+    Setting up Harbor
+    setenforce is /usr/sbin/setenforce
+    Setup Completed
+
+    Details:
+    -------
+    Spectro Cloud Repository
+    Location: https://palette.example.com:8443
+    UserName: spectro
+    Password: **************
+    CA certificate filepath: /opt/spectro/ssl/server.crt
+
+    Pack OCI Registry
+    Endpoint: https://palette.example.com
+    Base Content Path: spectro-packs
+    CA certificate Filepath: /opt/spectro/ssl/server.crt
+    Username: admin
+    Password: **************
+
+    Image OCI Registry
+    Endpoint: https://palette.example.comv
+    Base Content Path: spectro-images
+    CA certificate Filepath: /opt/spectro/ssl/server.crt
+    Username: admin
+    Password: **************
+    ```
+
+16. Update the SSL certificate file and key in the **httpd** service.
+
+    :::warning
+
+    You can skip this step if you do not plan to use the local Spectro Cloud Artifact Repository (SCAR) during the
+    [{props.edition} installation](#next-steps) process.
+
+    :::
+
+    Open the `/etc/httpd/conf.d/ssl.conf` file and add the path to the certificate and key generated in step **15** of
```

Comment:
@caroldelwing Is this file large? I'm mainly asking because it would be nice to show the changes through line highlights.

Reply:
Super large, like 200+ lines. I tried adding the snippet and it looked weird :(